.github/workflows/ci.yaml

```diff
@@ -74,9 +77,18 @@ jobs:
           python-version: ${{ matrix.python-version }}
           use-only-tar-bz2: true
 
+      - name: Cache conda env
+        id: cache-env
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.CONDA }}/envs
```

Comment:
Maybe 
```suggestion
          path: ${{ env.CONDA_PREFIX }}
```

`env.CONDA` is empty looking at the logs...

<img width="833" alt="image" src="https://user-images.githubusercontent.com/2448579/165668804-0abf5b7e-3890-4ec6-8336-1d5399021d99.png">

Reply:
Thanks. Weirdly it's listed in https://github.com/conda-incubator/setup-miniconda#caching-environments...

Reply:
Sorry, actually where do you see that it's empty?

Reply:
Ah, right, yes, `/envs`